dev/com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout/build.gradle

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+apply from: '../com.ibm.ws.security.saml.sso_fat.common/SAMLCommon.gradle'
+
+repositories {
+  if (isUsingArtifactory) {
+    maven {
+      credentials {
+        username userProps.getProperty("artifactory.download.user")
+        password userProps.getProperty("artifactory.download.token")
+      }
+      url ("https://" + gradle.userProps.getProperty("artifactory.download.server") + "/artifactory/wasliberty-maven-remote")
+    }
+  } else {
+    mavenCentral()
+  }
```

Comment:
Is this all needed?

Reply:
I copied what was done for com.ibm.ws.security.sso_fat.
I think I omitted it when I ported com.ibm.ws.security.sso_fat.2 and had problems - I can try to remove it (from this and possibly all of the other saml fats...)